integration-tests/bats/sql-merge.bats

```diff
@@ -782,33 +782,206 @@ SQL
     [ $status -eq 0 ]
 }
 
-@test "sql-merge: merging preserved check constraints" {
+# TODO: what happens when the data conflicts with new check?
```

Comment:
we should validate new data (we do this with FOREIGN KEY constraints), but it's outside the scope of this PR